lib/components/programs-list/index.tsx

```diff
@@ -19,42 +29,69 @@ const ProgramListing: React.FC<ProgramListingProps> = ({
   recordingLink,
 }) => {
   const breakpoint = useBreakpoint();
+  const editProgramModalManagement = useDisclosure();
   let useMobileLayout = breakpoint && PAGE_SIZES.includes(breakpoint);
 
   return (
-    <Flex flexDirection={useMobileLayout ? 'column' : 'row'} gap={4}>
-      <Flex flexDirection="column" alignItems={useMobileLayout ? 'start' : 'end'} minWidth={'40%'}>
-        <Heading textAlign={useMobileLayout ? 'left' : 'right'}>{date}</Heading>
-        <Text>{time}</Text>
-      </Flex>
-      <Box
-        h={useMobileLayout ? '4px' : 'maxContent'}
-        minWidth={useMobileLayout ? 'maxContent' : '4px'}
-        bg={'var(--magenta)'}
-      />
-      <Flex flexDirection="column" alignItems="start">
-        <Heading>{title}</Heading>
-        <Text fontSize="lg" whiteSpace="pre-line">
-          {description}
-        </Text>
-        <Link
-          href={recordingLink || registerLink}
-          isExternal
-          _hover={{ textDecoration: 'none' }}
-          _focus={{ outline: 'none' }}>
+    <>
+      <EditProgramModal modalManagement={editProgramModalManagement} />
+
+      <Flex flexDirection={useMobileLayout ? 'column' : 'row'} gap={4}>
```

Comment:
Consider breaking this down into smaller components with descriptive names?

Reply:
This is old code that just got moved around slightly, refactoring it is probs out of scope of the ticket

Reply:
gotcha, agreed